src/oneid/oneid-ecs-core/src/main/webui/Dangerfile.ts

```diff
@@ -1,7 +1,6 @@
 // import custom DangerJS rules
```

Comment:
Danger is actually not used here, I think we can remove it

Reply:
So i can remove the file entirelly?

Reply:
yes, and so the `danger-plugin-digitalcitizenship` as well

Reply:
I've removed the Dangerfile and dependencies